f600233d1b154d507b782f84604bcb12/update/sys_ws_query_parameter_56c2243adbb269508c045e8cd3961953.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_ws_query_parameter">
+    <sys_ws_query_parameter action="INSERT_OR_UPDATE">
+        <example_value/>
+        <name>sysparm_display_value</name>
+        <required>false</required>
```

Comment:
I think this parameter should either be marked required, or the implementation made a bit more defensive.

![image](https://github.com/dod-ccpo/atat-snow/assets/77642966/896270ec-4786-4a3e-8212-a0a68736b00b)

I was failing to get results the first few times I tried to use it because I left this empty.  For example:
![image](https://github.com/dod-ccpo/atat-snow/assets/77642966/2cca2bfd-277e-432d-a65e-f676d4934a76)

Reply:
Apart from the above, it works great when I include a value for `sysparm_display_value`